### DISCUSSION TITLE
Cannot deploy  node js github repo on vercel by using vercel.json file.

### DISCUSSION THREAD

**Discussion by Rivindhu** [vercel/community#1020] [Help] [answered]
I want to deploy  a github node js repository to vercel. And I created the "vercel.json" file. And I try to deploy the project on vercel, but it's gave me an error "undefined	ERROR	Cannot find module '/var/task/config/config.js'". But I checked the config.js file it is located in the folder called "config" and it's correctly defined in the "app.js" file. I tried many things nothing work for me yet. I need a help.

The project that I want to deploy is -> https://github.com/Rivindhu/rarity-analyser-02-11-2022

This project is not originally developed by me.
Original github repo -> https://github.com/fukuball/rarity-analyser

Thank You!

**Comment by amyegan** [comment, marked as answer]
Hi @Rivindhu! Does it work if you make the path relative to `app.js`, like `const config = require("./config/config.js");`?

**Reply by Rivindhu** [reply]
Thank you! Miss, for your time. 
I changed the app.js file as below,

`// const appRoot = require("app-root-path");`
`const config = require("./config/config");`

I commented out the "appRoot" variable and I explicitly defined the path of the config.js file.

Now it's says,
"undefined	ERROR	SqliteError: unable to open database file"

Still not work for me.

Thank you!

**Reply by amyegan** [reply]
😞 SQLite does not work well in serverless environments because storage is not permanent. You can find more details and suggested alternatives in ["Is SQLite supported in Vercel?"](https://vercel.com/guides/is-sqlite-supported-in-vercel)
